components/logging/org.wso2.carbon.logging.appender.http/src/main/java/org/wso2/carbon/logging/appender/http/utils/queue/PersistentQueueException.java

```diff
@@ -0,0 +1,37 @@
+package org.wso2.carbon.logging.appender.http.utils.queue;
```

Comment:
create exception package and have base, exteded into server and client exceptions

Reply:
The client exceptions are not generated inside the queue. Therefore need we may not need separate server and client exceptions.

Reply:
Decided to keep as it is after offline discussion.